lib/resources/r4/related_person_patch.yaml

```diff
@@ -126,6 +126,45 @@ operations:
         <li>When specifying a `period`, the `start` and `end` fields must include a time component with a timezone.</li>
       </ul>
 
+  - name: add-telecom
+    path: /telecom/-
+    operation: add
+    type: ContactPoint
+    url: https://hl7.org/fhir/R4/patient-definitions.html#Patient.telecom
+    description: Add a new contact detail for the related person.
+    example: |
+      [
+          {
+          	"op": "add",
+          	"path": "/telecom/-",
+          	"value": {
+                  "use": "home",
+                  "system": "email",
+                  "value": "test@test.com",
+                  "period": {
+                      "start": "2018-01-05T22:24:32.000Z"
+                  }
+              }
+          },
+          {
+          	"op": "add",
+          	"path": "/telecom/-",
+          	"value": {
+                  "use": "home",
+                  "system": "phone",
+                  "value": "913-444-8881",
+                  "period": {
+                      "start": "2018-01-05T22:24:32.000Z"
+                  }
+              }
+          }
+      ]
+    note: >
+      <ul>
+      <li>Must contain the `system`, `use` and `value` fields.</li>
+      <li>Must include a time component with a timezone when specifying a `period`.</li>
```

Comment:
Let's be a little more specific with this statement. Another fhir.cerner.com update has this statement about the `period`: https://github.com/cerner/fhir.cerner.com/pull/550/files#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR36

Reply:
Done here:https://github.com/cerner/fhir.cerner.com/pull/559/commits/c8b522d37a1128eb487d1f87a377214d97176251#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR165